src/lib/mixer_module/mixer_module.cpp

```diff
@@ -499,12 +499,7 @@ int MixingOutput::controlCallback(uintptr_t handle, uint8_t control_group, uint8
 	input = output->_controls[control_group].control[control_index];
 
 	/* limit control input */
-	if (input > 1.0f) {
-		input = 1.0f;
-
-	} else if (input < -1.0f) {
-		input = -1.0f;
-	}
+	math::constrain(input, -1.f, 1.f);
```

Comment:
@MaEtUgR this is quite a severe bug: `input = math::constrain(input, -1.f, 1.f);`

Reply:
Let me fix it 😬

Reply:
Thanks for taking a closer look again!! https://github.com/PX4/Firmware/pull/15379
I was convinced it's pass by reference 😞